DEPS

```diff
@@ -10,7 +10,7 @@ deps = {
   "vendor/boto": "https://github.com/boto/boto@f7574aa6cc2c819430c1f05e9a1a1a666ef8169b",
   "vendor/python-patch": "https://github.com/svn2github/python-patch@a336a458016ced89aba90dfc3f4c8222ae3b1403",
   "vendor/sparkle": "https://github.com/brave/Sparkle.git@c0759cce415d7c0feae45005c8a013b1898711f0",
-  "vendor/bat-native-ledger": "https://github.com/brave-intl/bat-native-ledger@a3e64797bf6ebbfa39a60240db8d82995f97ebd9",
+  "vendor/bat-native-ledger": "https://github.com/brave-intl/bat-native-ledger@123d1dec129528390eaf583d04e5dc2b6c426713",
```

Comment:
what was this bumped to?

Reply:
Bumped to same as what it was + this:
https://github.com/brave-intl/bat-native-ledger/commit/123d1dec129528390eaf583d04e5dc2b6c426713

Reply:
I see, I guess I had already removed the post task stuff but just forgot about the header at that point